app/assets/src/components/common/DetailsSidebar/TaxonDetailsMode/TaxonDetailsMode.jsx

```diff
@@ -184,7 +184,7 @@ export default class TaxonDetailsMode extends React.Component {
       this.state.histogramRpmSeries,
       {
         seriesNames: ["NT", "NR"],
-        labelX: "rpm",
+        labelX: "rpm+1",
```

Comment:
I don't really like `rpm+1`. It adds additional confusion for the user. I would try out the clamp approach and see if that works. The clamp approach will basically treat 0 the same as 1 (so it's impossible to tell them apart in the graph) but that should be fine for our purpose.

Reply:
@MarkAZhang Agree it is a bit confusing, but it is however, scientifically inaccurate... so I would run it through @katrinakalantar

Reply:
Just tested out reverting to `rpm` rather than `rpm+1` and using clamp, it looks like all the out of range values (<1) get dropped in `getBins()` and don't show up on the graph. Looking into that right now.

(This is the Ralstonia example used above.)
* **rpm+1, no clamp**
![image](https://user-images.githubusercontent.com/53838890/65637085-6db34900-df98-11e9-8f30-0f7c07a6e12e.png)

* **rpm, using clamp**
![image](https://user-images.githubusercontent.com/53838890/65637099-7441c080-df98-11e9-8d93-fa8f8342e745.png)

Reply:
It seems that when generating bins for the histogram, values outside the domain (taken from the x scale) are [ignored](https://github.com/d3/d3-array/blob/v1.2.4/README.md#histogram_domain). Doesn't seem like there's an equivalent clamp function to use for histograms :/

Reply:
The other question if if we want to get rid of rpm+1 (which is more correct)

Reply:
I think it's important to have rpm=0 represented on this graph, i.e. prefer top graph over bottom graph. Often, the whole point will be to see how many samples did not contain the microbe at all, and how unusual it is that the sample does contain it. So dropping the lowest bin as shown in the bottom graph is not an option

Reply:
It would be very misleading.

Reply:
> dropping the lowest bin as shown in the bottom graph is not an option

I believe the clamp idea is that the 0 rpm would actually return a value of 1 1 (so having 0 and 1 would be represented in the same bar). 
I guess the question is if people will understand the `rpm+1` label, which makes the chart correct.

Reply:
I see. I believe rpm+1 is fairly standard to deal with this issue in bioinformatics